src/sqlancer/Reproducer.java

```diff
@@ -1,5 +1,15 @@
 package sqlancer;
 
+import java.util.List;
+
 public interface Reproducer<G extends GlobalState<?, ?, ?>> {
     boolean bugStillTriggers(G globalState);
```

Comment:
Perhaps now, we can rename this method to `bugTriggers`, given that we create the reproducer also if no bug was triggered?